nodewatcher/modules/monitor/http/survey/cgm.py

```diff
@@ -0,0 +1,10 @@
+from nodewatcher.core.generator.cgm import base as cgm_base
+
+
+@cgm_base.register_platform_module('openwrt', 50)
+def clients(node, cfg):
+    """
+    Register packages needed for reporting the clients.
+    """
+
+    cfg.packages.update(['nodewatcher-agent-mod-clients'])
```

Comment:
CGMs (configuration generating modules) are functions, which generate firmware configuration. In this instance, you just specify that the `nodewatcher-agent-mod-clients` OpenWrt package should be installed for each node. However, I think that you use output from the `wireless` module and not `clients`. So you should change this to depend on the correct module.

The modules are those of [nodewatcher-agent](https://github.com/wlanslovenija/nodewatcher-agent/tree/master/modules), with packages defined in [our OpenWrt package feed](https://github.com/wlanslovenija/firmware-packages-opkg/blob/master/util/nodewatcher-agent/Makefile#L87).